lib/engine_wrapper.py

```diff
@@ -1068,15 +1068,15 @@ def get_lichess_egtb_move(li: LICHESS_TYPE, game: model.Game, board: chess.Board
                 dtm *= -1
             logger.info(f"Got move {move} from tablebase.lichess.ovh (wdl: {wdl}, dtz: {dtz}, dtm: {dtm}) for game {game.id}")
         else:  # quality == "suggest":
-            best_wdl = name_to_wld[data["moves"][0]["category"]]
+            best_wdl = name_to_wld[data["moves"][0]["category"]] * -1
 
             def good_enough(possible_move: LichessEGTBMoveType) -> bool:
-                return name_to_wld[possible_move["category"]] == best_wdl
+                return name_to_wld[possible_move["category"]] * -1 == best_wdl
 
             possible_moves = list(filter(good_enough, data["moves"]))
             if len(possible_moves) > 1:
                 move_list = [move["uci"] for move in possible_moves]
-                wdl = best_wdl * -1
+                wdl = best_wdl
```

Comment:
Am I wrong, or do these code changes result in no change in behavior?

Reply:
This doesn't change anything. I just though that the code would be easier to understand because the correct wdl is with the ` * -1`, but we can obviously test with the wrong wdl and correct it at the end as we are currently doing. The bug was in line 1105.